bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -161,7 +162,7 @@ public void refreshSource() {
     @Override
     public List<Addon> getAddons(Locale locale) {
         try {
-            List<DiscourseCategoryResponse> pages = new ArrayList<DiscourseCategoryResponse>();
+            List<DiscourseCategoryResponseDTO> pages = new ArrayList<DiscourseCategoryResponseDTO>();
```

Comment:
```suggestion
            List<DiscourseCategoryResponseDTO> pages = new ArrayList<>();
```

Reply:
I didn't change these because there are a lot of similar issues in the code. But I'll fix these two.